index.html

```diff
@@ -1 +1,152 @@
-hello world
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Epilogue:wght@200;400;500;600;700&family=IBM+Plex+Sans:ital,wght@0,100;0,200;0,300;0,400;0,700;1,200;1,400&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
+    </link>
+    <title>Soyeon Kwon's Resume</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div id="info">
+            <h1>Soyeon<br>Kwon</h1>
```

Comment:
The `br` element generally should be reserved for meaningful line breaks, like in an address or poem; see [MDN's docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br) for more info. Here, you should be able to delete it because the width of your name should be constrained by the width set on the surrounding `div` on line 15.